src/Uno.UWP/ApplicationModel/Chat/ChatMessageManager.Android.cs

```diff
@@ -41,17 +41,38 @@ private static Intent CreateIntent(string body, string[] recipients)
 			if (!string.IsNullOrWhiteSpace(body))
 			{
 				intent.PutExtra(SmsBodyExtra, body);
-			}
-					   
-			var recipientUri = string.Join(
-                RecipientsSeparator,
-                recipients.Select(r => Android.Net.Uri.Encode(r)));					   
+			}
+
+			var recipientUri = string.Join(
+				RecipientsSeparator,
+				recipients.Select(r => Android.Net.Uri.Encode(r)));
 			var uri = Android.Net.Uri.Parse($"{SmsProtocol}{recipientUri}");
 
 			intent.SetData(uri);
 
 			return intent;
 		}
+
+		public static IAsyncOperation<ChatMessageStore> RequestStoreAsync()
+			=> RequestStoreAsyncTask().AsAsyncOperation<ChatMessageStore>();
+
+		public static async Task<ChatMessageStore> RequestStoreAsyncTask()
```

Comment:
Why do you create a task version of that method? The `IAsyncOperation` one is the only one needed.

Reply:
I don't know how to use `await` in IAsyncOperation